test/views/businessName/CaptureBusinessNameViewSpec.scala

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.businessName
+
+import forms.BusinessNameForm._
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.twirl.api.Html
+import views.ViewBaseSpec
+import views.html.businessName.CaptureBusinessNameView
+
+class CaptureBusinessNameViewSpec extends ViewBaseSpec {
+
+  val injectedView: CaptureBusinessNameView = inject[CaptureBusinessNameView]
+
+  object Selectors {
+    val pageHeading = "#content h1"
+    val backLink = ".govuk-back-link"
+    val form = "form"
+    val businessNameField = "#business-name"
+    val continueButton = ".govuk-button"
+    val errorSummary = "#business-name-error"
+    val errorSummaryTitle = "#error-summary-title"
+    val errorSummaryLink = ".govuk-error-summary__list > li > a"
+    val businessNameFormGroup = "#content > article > form > div:nth-child(1)"
+    val paragraphOne = "#content p:nth-of-type(1)"
+    val paragraphTwo = "#content p:nth-of-type(2)"
+    val businessGuidanceLink = "#content p:nth-of-type(1) a"
+
+    def listItem(num: Int): String = s"#content > div > ul > li:nth-child($num)"
+  }
+
+  "Rendering the capture trading name page" when {
+
+    "the user is a principle entity" when {
+
+      "the form has no errors" should {
+
+        lazy val view: Html = injectedView(businessNameForm(testBusinessName).fill(testBusinessName))(user, messages, mockConfig)
+
+        lazy implicit val document: Document = Jsoup.parse(view.body)
+
+        "have the correct document title" in {
+          document.title shouldBe "Change the business name - Business tax account - GOV.UK"
+        }
+
+        "have a back link" which {
+
+          "should have the correct text" in {
+            elementText(Selectors.backLink) shouldBe "Back"
+          }
+
+          "should have the correct href" in {
+            element(Selectors.backLink).attr("href") shouldBe "mockManageVatOverviewUrl"
```

Comment:
Bit picky, but could you change this to use the mockConfig value instead of "mockManageVatOverviewUrl"